ios/Classes/ScreenPlugin.m

```diff
@@ -25,6 +25,7 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
   else if ([@"keepOn" isEqualToString:call.method]) {
     NSNumber *b = call.arguments[@"on"];
     [[UIApplication sharedApplication] setIdleTimerDisabled:b.boolValue];
+    result(nil);
```

Comment:
The major change of this PR. @clovisnicolas could you merge & release a update on pub?

Reply:
@clovisnicolas any chance you could review/merge this fix?